Jenkinsfile

```diff
@@ -42,6 +42,11 @@ def isPR() { env.CHANGE_URL != null }
 def fork() { env.CHANGE_FORK ?: "stan-dev" }
 def branchName() { isPR() ? env.CHANGE_BRANCH :env.BRANCH_NAME }
 
+def writeIndexHTML() {
+  "echo '<head>' >> index.html"
```

Comment:
You can use a function called "writeFile" here - that wasn't me making one up, haha.

Reply:
Lol oh whoops good to know. Do you want me to do the redirect method or should we use the subtree method? If subtree then I'll get rid of this